Evolve.Migrations.Helper/Program.cs

```diff
@@ -46,15 +49,15 @@ public static void Main(string[] args)
             ChangeCsProjFile(rootPath, filePath);
         }
 
-        private static bool ValidateArguments(IReadOnlyList<string> args)
+        internal static bool ValidateArguments(IReadOnlyList<string> args)
         {
             if (args.Count == 0 || args[0] == "--help")
             {
                 PrintHeaderMessage();
-                Console.WriteLine($"\r\nUsage: {Assembly.GetExecutingAssembly().GetName().Name} [command] [options]\r\n");
+                Console.WriteLine($"\nUsage: {Assembly.GetExecutingAssembly().GetName().Name} [command] [options]\n");
                 Console.WriteLine("Commands:");
                 Console.WriteLine($"    {"add-dataset".PadRight(20, ' ')} Add a new migration file (on path: \"./datasets\")");
-                Console.WriteLine($"    {"add-migration".PadRight(20, ' ')} Add a new migration file (on path: \"./migrations\")\r\n");
+                Console.WriteLine($"    {"add-migration".PadRight(20, ' ')} Add a new migration file (on path: \"./migrations\")\n");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/odair-pedro/evolve-migrations-helper/pullRequest?prid=5663793)